README.md

```diff
@@ -116,6 +116,8 @@ carthage update Box Result
 
 If the framework you want to add to your project has dependencies explicitly listed in a [Cartfile][], Carthage will automatically retrieve them for you. You will then have to **drag them yourself into your project** from the [Carthage/Build] folder.
 
+If the embedded framework in your project has dependencies to other frameworks you must list them in the [Cartfile][], and **link them to application target** (even if application target does not have dependency to that frameworks and never uses them).
```

Comment:
> If the embedded framework in your project has dependencies to other frameworks ~~you must list them in the Cartfile~~

Dependencies are brought in transitively. If your `Cartfile` depends on `A`, which depends on `a1` and `a2` you don't need to add `a1` and `a2` to `Cartfile`, just `A`.